iceoryx_posh/test/moduletests/test_roudi_mempool_introspection.cpp

```diff
@@ -226,12 +226,17 @@ TEST_F(MemPoolIntrospection_test, thread)
     // we use the hasSubscribers call to check how often the thread calls the send method
     mock->hasSubscribersReturn = false;
 
-    m_introspection.setSnapshotInterval(10);
+    using namespace iox::units::duration_literals;
+    iox::units::Duration timeout(100_ms);
```

Comment:
please rename this to `snapshotInterval`

Reply:
done